### DISCUSSION TITLE
How to connect unleash to server what does parameters mean?

### DISCUSSION THREAD

**Discussion by suprith-hub** [Unleash/unleash#6108] [General]
Hey frontend is good,
but in server side sdk im very confused: I have used python SDK
my code:
client = UnleashClient(
url="http://localhost:4242/api",
app_name='V11',
custom_headers={'Authorization': '*:development.76998a8514449e1b9b83b8ed631c7a9620489f189db0c54b05ff47c3'})

client.initialize_client()

Unleash Client feature fetch failed due to unexpected HTTP status code.
Unleash Client feature fetch failed due to exception: Unleash Client feature fetch failed!
Traceback (most recent call last):
File "C:\Users\supri\AppData\Local\Programs\Python\Python311\Lib\site-packages\UnleashClient\api\features.py", line 74, in get_feature_toggles
raise Exception(
Exception: Unleash Client feature fetch failed!

I am very confused, what is this app-name??? and what is the url ? url of backend server or unleash server and why nothing works

**Comment by nunogois** [comment]
Hi @Era-cell,

Like @sighphyre [mentioned before](https://github.com/orgs/unleash/discussions/6074#discussioncomment-8343041), all the information you need can be found in the docs: https://gh.getunleash.io/unleash-client-python/unleashclient

If you need further assistance, please try asking for help on our [community Slack](https://slack.unleash.run/).
